engine/common/sys_con.c

```diff
@@ -265,11 +265,19 @@ static void Sys_PrintStdout( const char *logtime, const char *msg )
 	IOS_Log( buf );
 #endif // TARGET_OS_IOS
 
-#if (XASH_NSWITCH && NSWITCH_DEBUG) || XASH_PSVITA // REMOVEME
+#if XASH_NSWITCH && NSWITCH_DEBUG
 	// just spew it to stderr normally in debug mode
 	fprintf( stderr, "%s %s", logtime, buf );
 #endif // XASH_NSWITCH && NSWITCH_DEBUG
 
+#if XASH_PSVITA
+	// spew to stderr only in developer mode
```

Comment:
Why though? Does writing to stderr even matters on this platform?

Then it could be shared with NSwitch code as well! It doesn't look PSVita-specific.

Reply:
Either make it always print, either let NSwitch print to stderr with enabled host_developer.

But we shouldn't depend on host_developer here, all the optional console messages are rejected even before this function gets called.

Reply:
> Why though? Does writing to stderr even matters on this platform?
> 
> Then it could be shared with NSwitch code as well! It doesn't look PSVita-specific.

Used to be slow, so I didn't want to make it always do that. Not sure about now. This can probably be restored to the old state then.

Reply:
🤔
Again, we shouldn't print too much if developer mode is turned off and if console was disabled... Which isn't true, `psvita` has DEFAULT_ALLOW_CONSOLE set to 1.

Maybe leave this code as is and do not turn it into a platform-specific mess, but disable DEFAULT_ALLOW_CONSOLE?